File: solve.py
def getValor(a,i,j):
    try:
        return a[i][j]
    except:
        return ""

def alternate(a):
    w = ''
    for i in range(0,4):
        for j in range(0,4):
            w += getValor(a,j,i)
    return w

File: solve.js
function getValor(a,i,j){
    try {
        if (a[i][j]){
            return a[i][j];
        }
        return "";
    }
    catch(err){
        return "";
    }
}

function alternate(a){
    w="";
    for(i=0; i<5; i++){
        for(j=0; j<5; j++){
            w = w + getValor(a,j,i);
        }
    }
}